Parquet/src/main/java/io/deephaven/parquet/ParquetFileReader.java

```diff
@@ -75,6 +77,56 @@ public ParquetFileReader(String filePath, SeekableChannelsProvider channelsProvi
         type = fromParquetSchema(fileMetaData.schema,fileMetaData.column_orders);
     }
 
+    private HashSet<String> columnsWithDictionaryUsedOnEveryDataPage = null;
```

Comment:
We usually don't overspecify return types/variable types, as convention in Java. Better for this to be `Set`, unless you need `HashSet`-specific methods or features.